EIPS/eip-3238.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 3238
+title: Vatnajoekull Difficulty Bomb Delay
+author: Afri Schoedon (@q9f)
+discussions-to: https://github.com/ethereum/EIPs/pull/3238
```

Comment:
`discussions-to` cannot be the PR itself.  A thread over at https://ethereum-magicians.org/ is preferred, but if you don't have or want to create an account there you can create a in issue in this repository for "Discussion of ..." and link to it here.

Reply:
OK

```suggestion
discussions-to: https://github.com/ethereum/EIPs/issues/3239
```